fritz.defaults.yaml

```diff
@@ -62,6 +62,8 @@ skyportal:
         component: Alerts
       - path: "/alerts/ztf/:id"
         component: ZTFAlert
+      - path: "/alerts_by_coords/:ra/:dec/:radius"
```

Comment:
I suggest we do `/alerts_by_coords/ztf` since what we have here is very ZTF-specific, and we may/will implement this for other surveys (e.g. PGIR). + use query parameters as discussed above. @dannygoldstein you agree?

Reply:
I still think we should do /alerts_by_coords/ztf/...